tests/testthat/test.csv

```diff
@@ -0,0 +1,101 @@
+,contNormal,contGamma,contBinom,contExpon,contWide,contNarrow,contOutlier,contcor1,contcor2,facGender,facExperim,facFive,facFifty,facOutlier,debString,debMiss1,debMiss30,debMiss80,debMiss99,debBinMiss20,debNaN,debNaN10,debInf,debCollin1,debCollin2,debCollin3,debEqual1,debEqual2,debSame
```

Comment:
So... every module is going to have a copy of the debug dataset?

I would do that a little bit differently.

perhaps:

1. a specific package for this (called`jaspDatasets`?). A very rigorous solution, but probably overkill.
2. make the debug data set live inside `jasptools`.

Reply:
jaspTools fetches all the regular datasets from jasp-desktop Resources, except for unit test specific ones.
I could include `test.csv` as well I guess, but I wasn't sure if people would want to edit this file.